xlml/utils/xpk.py

```diff
@@ -62,7 +62,7 @@ def run_workload(
       f"gcloud config set compute/zone {zone}",
       "git clone https://github.com/google/xpk.git /tmp/xpk",
       "cd /tmp/xpk",
-      "apt-get update && apt-get install -y google-cloud-sdk",
+      "apt-get install -y google-cloud-sdk",
```

Comment:
Wondering the reason to take off `apt-get update`?

Reply:
Good catch! I was meaning to add it back